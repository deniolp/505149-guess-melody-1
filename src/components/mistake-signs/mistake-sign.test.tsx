import * as React from 'react';
import * as renderer from 'react-test-renderer';
import MistakeSigns from './mistake-signs';

describe(`QuestionScreenHeader`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<MistakeSigns
      mistakes={2}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
