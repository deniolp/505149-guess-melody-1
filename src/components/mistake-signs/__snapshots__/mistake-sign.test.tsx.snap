// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionScreenHeader renders correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
